tests/ed25519.rs

```diff
@@ -124,7 +124,7 @@ fn test_eq_ord() -> crypto::Result<()> {
     assert!(pk == pk_eq);
     assert!(pk != pk_diff);
     assert!(pk > pk_diff);
-    assert!(!(pk > pk_eq));
+    assert!(pk <= pk_eq);
```

Comment:
A couple lines above there's `assert!(pk == pk_eq);` already, it's safe to remove this line `assert!(pk <= pk_eq);`